src/org/labkey/targetedms/query/PeptideManager.java

```diff
@@ -148,6 +148,7 @@ public static List<PeptideCharacteristic> getCombinedPeptideCharacteristics(long
         sql.append(" AND pep.StartIndex IS NOT NULL AND pep.EndIndex IS NOT NULL ");
         sql.append(" GROUP BY pep.Sequence,pci.SampleFileId, pep.StartIndex, pep.EndIndex ) X ");
         sql.append(" GROUP BY X.Sequence, X.StartIndex, X.EndIndex ");
+        sql.append(" ORDER BY X.Sequence ");
```

Comment:
@labkey-jeckels do you see any problems with the order by here?

Reply:
No, this seems fine to me.